hammr/utils/migration_utils.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2007-2017 UShareSoft SAS, All rights reserved
+#
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from texttable import Texttable
+
+
+def migration_table(migrations):
+    table = Texttable(800)
+    table.set_cols_dtype(["t", "t", "t"])
+    table.set_cols_align(["c", "l", "c"])
+    table.header(["Id", "Name", "Status"])
+
+    for migration in migrations:
+        status = migration.status.message + " (" + str(migration.status.percentage) + "%)"
+        if migration.status.complete:
```

Comment:
Maybe add if migration.status.error and cancel because we don't want to see the percentage, only message in these cases too